firefox_privacy_notice/en-US.md

```diff
@@ -35,7 +35,9 @@ Firefox displays content, such as “Snippets” (messages from Mozilla), Add-on
 * __Webpage data for Snippets__: When you choose to click on a Snippet link, we may receive data about the link you followed. This information is not associated with any other information about you. [Learn more](https://abouthome-snippets-service.readthedocs.io/en/latest/data_collection.html).
 {: #snippets }
 
-* __Webpage data for Pocket Recommendations__: We recommend content to you based on your browsing history. The process of deciding which stories you should see happens locally in your copy of Firefox and neither Mozilla nor Pocket (which is part of the Mozila family) receives a copy of your browsing history. We do receive aggregated data about the recommendations you see and click. This data does not identify you personally. We also share aggregated data with our third-party ad platform Adzerk about the sponsored content you see and click so advertisers can see how many people click on their articles, but not who clicked or viewed them.
+* __Webpage, Language, and Location data for Pocket Recommendations__: We recommend content to you based on your browsing history, language, and country location. The process of deciding which stories you should see based on your browsing history happens locally in your copy of Firefox, and neither Mozilla nor Pocket receives a copy of your browsing history. To help you see relevant Pocket Recommendations based on your location, Firefox shares your language and country location with Pocket.
+
+    Mozilla and Pocket receive aggregated data about the recommendations you see and click. We also share aggregated data about the sponsored content you see and    click with our third-party ad platform Adzerk so advertisers can see how many people click on their articles. This aggregated data does not identify you personally.
```

Comment:
Remove extra white space between `and` and `click with our third-party...`

Reply:
Good catch! It should be updated now